### ISSUE TITLE
Editing, deleting providers doesn't work

### ISSUE THREAD

**Issue by Cook66** [nosmokingbandit/watcher#184]
#### What happened?
Tried to edit and delete providers, after refreshing they stay the same.
#### What was supposed to happen?
Changes should be saved
#### How can I reproduce this?
Try editing a provider.

--
I'm using firefox

**Comment by jaketame** [comment]
I am also unable to add new providers... 

I believe I'm experiencing this issue, unable to add provider, tickbox is selected. I click Save Settings however doesn't do anything. Running Safari, no logs in console.

Log at same time to test connection, which seems to suggest nothing has been entered but it has.

ERROR 2017-01-21 20:59:13,379 core.newznab.test_connection: NewzNab connection check.
Traceback (most recent call last):
File "/opt/watcher/core/newznab.py", line 163, in test_connection
response = ET.fromstring(xml).attrib
File "", line 125, in XML
ParseError: no element found: line 1, column 0

**Comment by t4nkw4rt** [comment]
what's the indexer syntax in config.cfg?
maybe I can add them using a text editor

**Comment by soundjester** [comment]
generally, it is:

```
[Indexers]
1 = https://address/,APIKEY,true
2 = https://address/,APIKEY,true
etc...
```

**Comment by t4nkw4rt** [comment]
that worked. thanks!

**Comment by nosmokingbandit** [comment]
Fixed in commit 42363d7fb1639069ec3a8fcfa0be6fd3efb71923.
